.coveragerc

```diff
@@ -81,6 +81,7 @@ omit =
     homeassistant/components/climate/sensibo.py
     homeassistant/components/climate/touchline.py
     homeassistant/components/climate/venstar.py
+    homeassistant/components/climate/xs1.py
```

Comment:
Please move all the xs1 platforms under the xs1 component package.

Reply:
done